include/nova_renderer/util/utils.hpp

```diff
@@ -34,6 +34,16 @@ namespace nova::renderer {
 
     bool ends_with(const std::string& string, const std::string& ending);
 
+    template<typename Arg>
+    uint32_t va_count(const Arg& arg) {
+        return 1;
+    }
+
+    template<typename Arg, typename... Args>
+    uint32_t va_count(const Arg arg, const Args&... args) {
+        return 1 + va_count(args);
```

Comment:
should be:

```cpp
template<typename... Args>
uint32_t va_count(const Args&... args) {
    return sizeof...(args);
}
```

Reply:
Tbh not sure this function needs to exist

Reply:
It actually doesn't. I thought I could be cool and make a variadic template to build a path out of an arbitrary number of strings, but then msvc hit the template recursion depth and I never tried ever again